### PR TITLE
fix(lang): Add missing Arabic translations

### PR DESCRIPTION
**Pull request by misteroneill**
Updates Arabic translations. Some appear to be incomplete, but this how they were returned from our translation service.

### CODE REVIEW

File: lang/ar.json [hunk 1]
```diff
@@ -30,5 +30,58 @@
...
+  "Raised": "بارز",
+  "Depressed": "منخفض",
+  "Uniform": "منتظم",
+  "Dropshadow": "ظل خلفي",
+  "Font Family": "عائلة الخطوط",
+  "Proportional Sans-Serif": "Proportional Sans-Serif",
+  "Monospace Sans-Serif": "Monospace Sans-Serif",
+  "Proportional Serif": "Proportional Serif",
+  "Monospace Serif": "Monospace Serif",
+  "Casual": "Casual",
```
Comment (mister-ben) [inline lang/ar.json]: Are these left untranslated because they don't relate to Arabic typography?

File: lang/ar.json [hunk 2]
```diff
@@ -30,5 +30,58 @@
...
   ", opens subtitles settings dialog": ", تفتح نافذة  خيارات الترجمة",
-  ", selected": ", مختار"
+  ", selected": ", مختار",
+  "Audio Player": "مشغل الصوت",
+  "Video Player": "مشغل الفيديو",
+  "Replay": "إعادة التشغيل",
+  "Seek to live, currently behind live": "ذهاب إلى نقطة البث المباشر، متأخر عن البث المباشر حاليًا",
+  "Seek to live, currently playing live": "ذهاب إلى نقطة البث المباشر، البث المباشر قيد التشغيل حاليًا",
+  "Progress Bar": "شريط التقدم",
+  "progress bar timing: currentTime={1} duration={2}": "توقيت شريط التقدم: currentTime={1} duration={2}",
```
Comment (gkatsev) [inline lang/ar.json]: This line is incorrectly translated. The translation should read something like `{1} of {2}` or something.

File: lang/ar.json (hunk 1 above)
Comment (misteroneill) [reply lang/ar.json]: That's how they came back from the translation service.

File: lang/ar.json (hunk 2 above)
Comment (misteroneill) [reply lang/ar.json]: Oh, right. 🤔 

I wonder if we should just do `{1} / {2}` for now while we work on getting an updated translation?

Comment (gkatsev) [reply lang/ar.json]: Might be better to just leave it off for now.

**Pull request merged**